control-plane/pkg/reconciler/consumergroup/consumergroup_test.go

```diff
@@ -144,8 +147,9 @@ func TestReconcileKind(t *testing.T) {
 							{PodName: "p1", VReplicas: 1},
 							{PodName: "p2", VReplicas: 1},
 						}
-						_ = cg.MarkReconcileConsumersFailed("PropagateSubscriberURI", ErrNoSubscriberURI)
+						cg.MarkReconcileConsumersSucceeded()
```

Comment:
Because of adding [this](https://github.com/knative-sandbox/eventing-kafka-broker/pull/2615/files#diff-5fe171b24257dcfc348ae476d5c41bd4e9e04f7a4003865adc523df2c2d6e05aR123) check for total replicas around the SubscriberURI status - total replica count for `consumers` is 0 for these unit tests since we don't add New Consumers to the [Lister](https://github.com/aavarghese/eventing-kafka-broker/blob/0a44fc2522e04667aa4d685b1f79fe890cd04ee5/control-plane/pkg/reconciler/consumergroup/consumergroup.go#L371)...